server/viame_server/transforms.py

```diff
@@ -58,9 +58,9 @@ def transform(self):
 
         return self.folder_path
 
-    # def cleanup(self):
-    #     shutil.rmtree(os.path.dirname(self.folder_path),
-    #                   ignore_errors=True)
+    def cleanup(self):
+        shutil.rmtree(os.path.dirname(self.folder_path),
+                      ignore_errors=True)
```

Comment:
Was this just causing errors prior?

Reply:
I think the previous author wanted to log into the container to look at the leftovers, so they commented this out.  I uncommented because any created file in the worker needs to be removed afterward.